roles/coreos-bootstrap/tasks/main.yml

```diff
@@ -1,41 +1,40 @@
 ---
```

Comment:
don't move the file from preinstall, do a different PR to cleanup if not required by other OS